message_sender/tasks.py

```diff
@@ -212,6 +209,13 @@ def run(self, message_id, **kwargs):
             l.info("Attempts: %s" % message.attempts)
             # send or resend
             try:
+                if not message.channel:
+                    channel = Channel.objects.get(default=True)
+                else:
+                    channel = Channel.objects.get(channel_id=message.channel)
```

Comment:
If we make the foreign key change above, then this can just be `channel = message.channel`.